src/Sugar.ml

```diff
@@ -301,12 +301,15 @@ let rec functor_type cmts ~for_functor_kw ({ast= mty} as xmty) =
 (* The sugar is different when used with the [functor] keyword. The syntax
    M(A : A)(B : B) cannot handle [_] as module name. *)
 let rec functor_ cmts ~for_functor_kw ({ast= me} as xme) =
+  let valid_sugared_name arg =
+    Migrate_parsetree.Versions.OCaml_current.version >= 408
```

Comment:
Formatting a project in a 4.08 switch could make it not build with previous versions..
I don't think we should take `Ocaml_current.version` into account here.

Reply:
@jberdine, any suggestions on how to handle this case ?

Reply:
My hunch is that we should have a config for that kind of things where one can mention a minimum version that needs to be supported.

Reply:
I changed to preserve the current syntax if an argument is `_` and no longer depend on the version of the parsetree. What do you think?